src/AppKit/Enums.cs

```diff
@@ -2277,7 +2277,7 @@ public enum NSPopUpArrowPosition : ulong {
 	}
 
 	// FileType 4cc values to use with NSFileTypeForHFSTypeCode.
-	[NoMacCatalyst]
+	[MacCatalyst(15, 0)]
```

Comment:
Hmm, not just your entry in this file, but I am wondering why this entire file is not using the dotnet attributes and why the test for the dotnet attributes is not going off?

Reply:
I'm not sure, I don't have any background on the dotnet attributes / traditional ones.

@mandel-macaque / @rolfbjarne - ?

Reply:
It's because we're treating this file as an API definition file, so the generator handles it:

https://github.com/xamarin/xamarin-macios/blob/09905e80211dec4250a8e9a3701808b9aa457f73/src/frameworks.sources#L79-L81

Reply:
That was my (semi-educated) guess. Glad for confirmation!